module/main.c

```diff
@@ -532,13 +508,10 @@ static void handler_HidDisconnect(s32 data) {
 // keys
 
 static void handler_HidTimer(s32 data) {
-    u8 i, n;
-
-    const s8* frame;
     if (hid_get_frame_dirty()) {
-        frame = (const s8*)hid_get_frame_data();
+        const s8* frame = (const s8*)hid_get_frame_data();
 
-        for (i = 2; i < 8; i++) {
+        for (size_t i = 2; i < 8; i++) {
```

Comment:
was going to ask why size_t is a better choice here (and in other loops) but then realized that it makes perfect sense! :+1: